go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -681,22 +681,20 @@ func (n *NodeController) syncFlows() {
 		line = strings.TrimSpace(line)
 		cookie := strings.TrimPrefix(strings.Split(line, ",")[0], "cookie=0x")
 		if cacheEntry, ok := n.flowCache[cookie]; ok {
-			// we only ever have one learned flow per pod, so if we already cached it, we dont need to do it again
-			if cacheEntry.learned {
-				klog.V(5).Infof("Ignoring flow to add to hybrid cache as it is already learned: %s", line)
-				continue
-			}
 			// we ignore certain cookies for learning to avoid a case where a NS was updated with a new vtep
 			// and we accidentally pick up the old vtep flow and cache it. This should only ever happen on a pod update
 			// with an NS annotation VTEP change. We only need to ignore it for one iteration of sync.
 			if cacheEntry.ignoreLearn {
 				klog.V(5).Infof("Ignoring learned flow to add to hybrid cache for this iteration: %s", line)
 				cacheEntry.ignoreLearn = false
+				cacheEntry.learnedFlow = ""
```

Comment:
is `ignoreLearn` still relevant?

Reply:
Yeah because we will resync flows again and we need to make sure we learn the flow for this cache entry the next time by resetting this to false.